src/manifolds/PowerManifold.jl

```diff
@@ -1,17 +1,45 @@
+"""
```

Comment:
We should/could accompany the types by concrete examples how to generate such data, maybe here or in `power.md` or in the two occasions `GraphManifold` and `Torus` and link to those from here?

Reply:
OK, I'll add examples for graph manifold and torus and link it from here.